src/RepoUtil/project.json

```diff
@@ -0,0 +1,21 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "version": "1.0.0-rc2-3002702",
+      "type": "platform"
```

Comment:
Believe this part is new.  Does this make the tool a .Net Core app?  Will it still run on Desktop CLR?